build_mime_types/build_mime_types.py

```diff
@@ -0,0 +1,226 @@
+import csv
+import os
+import re
+from inspect import getsourcefile
+from os.path import abspath
+from string import Template
+from typing import List, Dict, Final, Optional, Any, Union
+
+INT_MAP: Final = {
+    0: "ZERO",
+    1: "ONE",
+    2: "TWO",
+    3: "THREE",
+    4: "FOUR",
+    5: "FIVE",
+    6: "SIX",
+    7: "SEVEN",
+    8: "EIGHT",
+    9: "NINE"
+}
+
+CSV_FILES: Final = {
+    "application.csv",
+    "audio.csv",
+    "font.csv",
+    "image.csv",
+    "message.csv",
+    "model.csv",
+    "multipart.csv",
+    "text.csv",
+    "video.csv"
+}
+
+# Mime types supported by OpenAPI but not IANA (https://swagger.io/docs/specification/media-types/)
+OPENAPI_SPECIFIC_MIME_TYPES: Final = {
+    "*/*": "ANY",
+    "application/*": "APPLICATION_ANY",
+    "application/*+json": "JSON_ANY",
+    "audio/*": "AUDIO_ANY",
+    "font/*": "FONT_ANY",
+    'image/*': "Image",
+    "message/*": "MESSAGE_ANY",
+    "model/*": "MODEL_ANY",
+    "multipart": "MULTIPART_ANY",
+    "text/*": "TEXT_ANY",
+    "video/*": "VIDEO_ANY",
+    'text/json': "JSON_TEXT",  # This isn't an OpenAPI specific mime type but it is needed as it already is in use
```

Comment:
FWIW: This content type is deprecated. We may want to mark as such so it can be removed later?

https://mimetype.io/text/json

Reply:
Sounds good to me 👍